sources/main.cpp

```diff
@@ -146,6 +155,11 @@ int main(int argc, char** argv) {
         boost::posix_time::ptime t1 = boost::posix_time::second_clock::local_time();
         boost::posix_time::time_duration diff = t1 - t0;
         std::cout << "Simulation finished in " << diff.total_milliseconds()/1000 << "s" << std::endl;
+      } else if (simulationType == "CTC") {
+        launchCriticalTimeCalculation(inputFile, outputFile, directory, nbThreads);
+        boost::posix_time::ptime t1 = boost::posix_time::second_clock::local_time();
+        boost::posix_time::time_duration diff = t1 - t0;
+        std::cout << "Simulation finished in " << diff.total_milliseconds()/1000 << "s" << std::endl;
```

Comment:
Please update the message (should be "critical time calculation finished in ...")